mediasms.go

```diff
@@ -139,10 +139,28 @@ func (c Client) GetStatus(messageID string) (models.APIResponse, error) {
 // ReplaceMessageBodyURLs removes urls and replaces them with mediasms acceptable values
 func ReplaceMessageBodyURLs(messageBody string, allURLs []string) string {
 	urlReplacements := []string{"{URL}", "{URL2}", "{URL3}", "{URL4}"}
-
-	for i := range allURLs {
-		messageBody = strings.Replace(messageBody, allURLs[i], urlReplacements[i], 1)
+	
+	splitBody := strings.Split(messageBody, " ")
+	
+	urlHashmap := make(map[string]bool)
+	
+	for _, url := range allURLs {
+		urlHashmap[url] = true
+	}
+	
+	replacedMessageBody := []string{}
+	
+	replacementURLIndex := 0
+	
+	for _, w := range splitBody {
+		if _, ok := urlHashmap[w]; ok {
```

Comment:
If `allUrls` contains 4 items, but the `messageBody` has 5 urls, this will return an `index out of range` error